src/tools/Rubble/Expand-RubbleArchive.ps1

```diff
@@ -11,6 +11,9 @@ The path to the ZIP file.
 .PARAMETER OutputLocation
 The path where to extract the content of the file.
 
+.PARAMETER FileFilter
+If the file filter is not a null string, it is interpreted as a list of one or more regular expressions which is tested against each file name.
```

Comment:
can you leave a more verbose explanation on what the filter really does? I know, it's extracting the file if it matches, it's just not very explicitly mentioned here, IMO :)